### PR TITLE
Improvement/linting

### LINKED ISSUE #1: Infra: Add CI

### LINKED ISSUE #2: Infra: Add lint
- [ ] Add lint
- [ ] Add prettier (might just be part of lint) (if desired) (?)
- [ ] Add commands to `package.json:scripts.check`

### PR DESCRIPTION
**Pull request by boygirl**
@ryan-roemer this PR adds linting and prettier and sets up travis CI

closes #1 
closes #2

### CODE REVIEW

**Comment by ryan-roemer** [comment]
Just integrated travis. Closing and reopening to hopefully kick travis.

File: .travisrc.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+language: node_js
```
Comment (ryan-roemer) [inline .travisrc.yml]: I think this needs to be `.travis.yml`

File: .travisrc.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+node_js:
+  - "8"
+  - "10"
+  - "11"
```
Comment (ryan-roemer) [inline .travisrc.yml]: Let's switch to `12` which is released.

File: .travisrc.yml (hunk 1 above)
Comment (boygirl) [reply .travisrc.yml]: oops, you're right

File: .travisrc.yml (hunk 2 above)
Comment (boygirl) [reply .travisrc.yml]: so 10, 11, 12? or 8, 10, 12?

Comment (ryan-roemer) [reply .travisrc.yml]: 8, 10, 12, aka the "official releases". We only do odd-numbered unstable releases when they're out and close enough to official even status to get a heads up on...

**Review by ryan-roemer — APPROVED** [review]
🎉

**Pull request merged**